CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.2.1]
+### Changed
+- **BREAKING:** Removed support for Node v12 in favor of v14 ([#137](https://github.com/MetaMask/eth-json-rpc-middleware/pull/137))
```

Comment:
Same as what Erik said in https://github.com/MetaMask/eth-sig-util/pull/263 — you can either put this entry (plus `### Changed`) underneath `## [Unreleased]` or you can leave updating the changelog to the end when you cut a new release if you like.

Reply:
I believe I have now made the necessary changes!